docs/aws-config.md

```diff
@@ -138,41 +134,29 @@ For more information, see [the AWS documentation on IAM users][14].
 
-## Credentials and configuration
-
-In the Velero directory (i.e. where you extracted the release tarball), run the following to first set up namespaces, RBAC, and other scaffolding. To run in a custom namespace, make sure that you have edited the YAML files to specify the namespace. See [Run in custom namespace][0].
 
-```bash
-kubectl apply -f config/common/00-prereqs.yaml
-```
+## Install and start Velero
 
-Create a Secret. In the directory of the credentials file you just created, run:
+Install Velero, including all prerequisites, into the cluster and start the deployment. This will create a namespace called `velero`, and place a deployment named `velero` in it.
 
 ```bash
-kubectl create secret generic cloud-credentials \
-    --namespace <VELERO_NAMESPACE> \
-    --from-file cloud=credentials-velero
+velero install \
+    --bucket <YOUR_BUCKET> \
+    --provider aws \
+    --secret-file ./credentials-velero
+    --backup-location-config region=<YOUR_REGION> \
+    --snapshot-location-config region=<YOUR_REGION> \
+    --secret-file ./credentials-velero
 ```
 
-Specify the following values in the example files:
-
-* In `config/aws/05-backupstoragelocation.yaml`:
-
-  * Replace `<YOUR_BUCKET>` and `<YOUR_REGION>` (for S3 backup storage, region is optional and will be queried from the AWS S3 API if not provided). See the [BackupStorageLocation definition][21] for details.
-
-* In `config/aws/06-volumesnapshotlocation.yaml`:
+Additionally, you can specify `--use-restic` to enable restic support, and `--wait` to wait for the deployment to be ready.
 
-  * Replace `<YOUR_REGION>`. See the [VolumeSnapshotLocation definition][6] for details.
+For more complex installation needs (such as changing the namespace), use either the Helm chart, or add `--dry-run -o yaml` options for generating the YAML representation for the installation.
 
-* (Optional, use only to specify multiple volume snapshot locations) In `config/aws/10-deployment.yaml` (or `config/aws/10-deployment-kube2iam.yaml`, as appropriate):
 
-  * Uncomment the `--default-volume-snapshot-locations` and replace provider locations with the values for your environment.
+## Setting AWS_CLUSTER_NAME
 
-* (Optional) If you run the nginx example, in file `config/nginx-app/with-pv.yaml`:
-
-    * Replace `<YOUR_STORAGE_CLASS_NAME>` with `gp2`. This is AWS's default `StorageClass` name.
-
-* (Optional) If you have multiple clusters and you want to support migration of resources between them, in file `config/aws/10-deployment.yaml`:
+* (Optional) If you have multiple clusters and you want to support migration of resources between them, you can use `kubectl edit deploy/velero -n velero` to edit your deployment:
 
     * Uncomment the environment variable `AWS_CLUSTER_NAME` and replace `<YOUR_CLUSTER_NAME>` with the current cluster's name. When restoring backup, it will make Velero (and cluster it's running on) claim ownership of AWS volumes created from snapshots taken on different cluster.
```

Comment:
Uncommenting won't be quite right - they'll have to add it in the appropriate place